### ISSUE TITLE
Button: Icons should optionally not be read by screen-readers

### ISSUE THREAD

**Issue by EldRoos** [seb-oss/green#2011]
### Description

The icon names are read when you focus a button with a screen-reader, often that is not needed (or wrong) when the button also have text. So it should be an option if the icon should have a aria-label.

Eg: Similar to this is read by a screen reader: "graphics arrow-left button Go back!".

### Who will benefit from this feature?

All that uses screen-readers.

**Comment by EldRoos** [comment]
Ping Lindstedt, Nils in teams when this is fixed.

**Comment by EldRoos** [comment]
Fixed.
